src/test/java/net/snowflake/client/core/SessionUtilLatestIT.java

```diff
@@ -390,4 +390,79 @@ public void testOktaAuthGetFail() throws Throwable {
       assertEquals(SqlState.IO_ERROR, e.getSQLState());
     }
   }
+
+  private SFLoginInput createOktaLoginInput() {
+    SFLoginInput input = new SFLoginInput();
+    input.setServerUrl("https://testauth.okta.com");
+    input.setUserName("MOCK_USERNAME");
+    input.setPassword("MOCK_PASSWORD");
+    input.setAccountName("MOCK_ACCOUNT_NAME");
+    input.setAppId("MOCK_APP_ID");
+    input.setOCSPMode(OCSPMode.FAIL_OPEN);
+    input.setHttpClientSettingsKey(new HttpClientSettingsKey(OCSPMode.FAIL_OPEN));
+    input.setLoginTimeout(1000);
+    input.setSessionParameters(new HashMap<>());
+    input.setAuthenticator("https://testauth.okta.com");
+    return input;
+  }
+
+  // Testing retry with Okta calls the service to get a new unique token. This is valid after
+  // version 3.15.0.
```

Comment:
We already released version 3.15.1 - this test will work in the versions above - please change the comment